keys.schema.json

```diff
@@ -126,6 +129,163 @@
         }
       },
       "required": ["packageId", "csrftoken", "sessionid"]
+    },
+    "Safari": {
+      "type": "object",
+      "properties": {
+        "zip": {
+          "type": "string"
+        },
+        "bundleId": {
+          "type": "string",
+          "description": "i.e. com.plasmo.mock"
+        },
+        "extensionBundleId": {
+          "type": "string",
+          "description": "i.e. com.plasmo.mock.extension"
+        },
+        "appName": {
+          "type": "string",
+          "description": "i.e. Plasmo Mock"
+        },
+        "appCategory": {
+          "type": "string",
+          "description": "last component of LSApplicationCategoryType (i.e. business)"
+        },
+        "platforms": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "defaults to iOS and macOS"
+        },
+        "buildNumber": {
+          "type": "number",
+          "description": "defaults to GITHUB_RUN_NUMBER"
+        },
+        "appleId": {
+          "type": "string",
+          "description": "Your Apple email address"
+        },
+        "appleDevPortalId": {
+          "type": "string",
+          "description": "Apple Developer Account email address"
+        },
+        "teamName": {
+          "type": "string",
+          "description": "i.e. Plasmo Corp."
+        },
+        "teamId": {
+          "type": "string",
+          "description": "i.e. Q2CBPJ58CA"
+        },
+        "itunesConnectId": {
+          "type": "string",
+          "description": "App Store Connect Account email address"
+        },
+        "itcTeamName": {
+          "type": "string",
+          "description": "i.e. \"Company Name\" for Apple IDs in multiple teams"
+        },
+        "itcTeamId": {
+          "type": "string",
+          "description": "i.e. \"18742801\" for Apple IDs in multiple teams"
+        },
+        "provisioningProfiles": {
```

Comment:
Can this be constructed from the other piece of data? Or is the idea that user can either providing a provisionProfiles array VS providing the granular detail above as singular fields?

Reply:
Since it seems to me the data in this profile object is the same as the data gathered so far above?